analyzers/rspec/cs/S2094_c#.html

```diff
@@ -14,9 +14,10 @@ <h2>Compliant Solution</h2>
 }
 </pre>
 <h2>Exceptions</h2>
-<p>Partial classes are ignored entirely, as they are often used with Source Generators. Subclasses of System.Exception are be ignored, as even an
-empty Exception class can provide useful information by its type name alone. Subclasses of certain framework types - like the PageModel class used in
-ASP.NET Core Razor Pages - are also be ignored.</p>
+<p>Partial classes are ignored entirely, as they are often used with Source Generators. Subclasses of System.Exception are ignored, as even an empty
```

Comment:
`are also be ignored.` --> "are also ignored"

Reply:
Fixed.